book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def initialize(title:, author:)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] You still need to include the method for the has-many-through relationship. Kindly do that.

  ```
  def add_rental(person)
    Rental.new(person, self)
  end
  ```